src/main/java/io/leonis/subra/game/data/Player.java

```diff
@@ -109,8 +127,11 @@ public State(
     }
   }
 
+  /**
+   * The {@link io.leonis.zosma.game.Identity identity} of the robot.
+   */
   @Value
-  class Identity implements Agent {
+  class Identity implements io.leonis.zosma.game.Identity {
```

Comment:
I think `PlayerIdentity` is a better name and prevents name clashing.

Reply:
`Player.PlayerIdentity`?

Reply:
Just import the packageso you can refer to `PlayerIdentity` directly.